README.md

```diff
@@ -1,2 +1,47 @@
-# community
-Istio Community events and related information such as meeting notes, etc.
+# Istio Community
+
+Welcome to the Istio community!
+
+This is the starting point for becoming a contributor - improving docs, improving code, giving talks etc.
+
+# Table of Contents
+
+1. [Community Meeting](#community-meeting)
+2. [Working Groups](#working-groups)
+3. [How can I help?](#how-can-i-help-)
+4. [General Developer Communication Information](communication.md)
+
+## Community Meeting
+
+We have PUBLIC and RECORDED [bi-weekly community meetings](https://zoom.us/j/986657835) every Thursday at 11am US Pacific Time.
```

Comment:
Not sure if the meeting ID is stable or not, I'll ask April and circle back. We should link to the recorded meetings too, but again I'll verify where they are and send a PR.